### PR TITLE
Add PrestoQueryRunner Document

### PR DESCRIPTION
**Pull request by duanmeng**

### CODE REVIEW

**Comment by duanmeng** [comment]
@mbasmanova @kgpai Could you please help to review this PrestoQueryRunner doc?

File: velox/exec/tests/utils/PrestoQueryRunner.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# How To Use PrestoQueryRunner
+
+Currently, Aggregation Fuzzer verifies results against DuckDB. However, not all
+functions are available in DuckDB and sometimes semantics don't match. It would
+be nice to use PrestoJava as a source of truth to verify test results.
+
+We've added PrestoJava to Velox in [#7628](https://github.com/facebookincubator/velox/pull/7628),
+and we can use `PrestoQueryRunner` instead of `DuckQueryRunner` now.
+
+**Note: Note: We currently need to change the code to use it. Once `PrestoQueryRunner`
```
Comment (kgpai) [inline velox/exec/tests/utils/PrestoQueryRunner.md:10]: nit: Remove one Note.

File: velox/exec/tests/utils/PrestoQueryRunner.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+works well, we will always use it and remove `DuckQueryRunner`.**
```
Comment (kgpai) [inline velox/exec/tests/utils/PrestoQueryRunner.md:11]: maybe we can provide which to use as source of truth / Duckdb or Presto .

File: velox/exec/tests/utils/PrestoQueryRunner.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+The following are the steps on how to use `PrestoQueryRunner`, you can also refer to
+the [PrestoQueryRunnerTest](https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/PrestoQueryRunnerTest.cpp).
+
+- Launch a Presto all-in-one instance or cluster, e.g. HiveQueryRunner in Presto Java
+- Make code change in [AggregationFuzzerTest](https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/AggregationFuzzerTest.cpp),
+remove the `duckQueryRunner` variable and new a `PrestoQueryRunner` variable named `prestoQueryRunner`.
```
Comment (kgpai) [inline velox/exec/tests/utils/PrestoQueryRunner.md:18]: and add a new `PrestoQueryRunner`..

File: velox/exec/tests/utils/PrestoQueryRunner.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- Use the `prestoQueryRunner` in the `main` method.
+
+# Detail Code diff
+
+Code diffs as the follow,
```
Comment (kgpai) [inline velox/exec/tests/utils/PrestoQueryRunner.md:23]: The code diff is as follows..

**Review by kgpai — COMMENTED** [review]
Looks Great, minor nits. 
I will start merge process as soon as you make the small changes.

**Comment by duanmeng** [comment]
@kgpai Hi Krishna, thanks for your review, I've resolved all the comments except for this one, I may not get your point precisely. Could you help take another look?

>maybe we can provide which to use as source of truth / Duckdb or Presto .

I added this **note** to highlight that Presto would be the only source of truth, and DuckDB would be removed, according to this comment (https://github.com/facebookincubator/velox/pull/7628#discussion_r1400563473) of the `PrestoQueryRunner` PR.

**Pull request closed without merging**